src/hotspot/share/c1/c1_Canonicalizer.cpp

```diff
@@ -548,6 +548,19 @@ void Canonicalizer::do_Intrinsic      (Intrinsic*       x) {
     }
     break;
   }
+  case vmIntrinsics::_getModifiers           : {
```

Comment:
Although old code below has this weird style, please change this to `_getModifiers: {`